src/components/table-transactions/table-transactions.scss

```diff
@@ -0,0 +1,50 @@
+.table-transactions {
+  border: 1px solid #ddd;
+  border-collapse: collapse;
+  width:100%;
+  margin:1%;
+  min-width: 600px;
+  text-align: left;
+  box-sizing: border-box;
+  font-family: $base-font-family;
```

Comment:
no sure that you need to set font family (we have got now enabled roboto by all app by default)